index/openrct2.toml

```diff
@@ -1,11 +1,11 @@
 name = "OpenRCT2"
 home = "https://discord.com/channels/731205301247803413/1095746758774108240"
-default_url =  "https://github.com/Crazycolbster/rollercoaster-tycoon-randomizer/releases/download/v.{{version}}/openrct2.apworld"
+default_url =  "https://github.com/Crazycolbster/rollercoaster-tycoon-randomizer/releases/download/v{{version}}/openrct2.apworld"
 
 [versions]
-"0.1.13-beta" = { url = "https://github.com/Crazycolbster/rollercoaster-tycoon-randomizer/releases/download/0.1.13-beta/openrct2.apworld" }
-"0.1.15-beta" = { url = "https://github.com/Crazycolbster/rollercoaster-tycoon-randomizer/releases/download/v0.1.15-beta/openrct2.apworld" }
-"0.1.16-beta" = { url = "https://github.com/Crazycolbster/rollercoaster-tycoon-randomizer/releases/download/v0.1.16-beta/openrct2.apworld" }
+"0.1.13-beta" = {}
```

Comment:
Lovely, those are all reuploads annd none of the checksums match. Please remove everything except for the last version. I'm tired of this